.gitignore

```diff
@@ -1,6 +1,8 @@
 node_modules
 .DS_store
 
+mise.toml
+
```

Comment:
I'd rather not include things in the gitignore for tools the stack does not officially use/recommend. Could this be in your global gitignore maybe?


```suggestion
```

Reply:
oops my bad! sorry about that.